apps/ui/src/views/Proposal.vue

```diff
@@ -116,7 +118,18 @@ watch(
     await proposalsStore.fetchProposal(props.space.id, id, props.space.network);
 
     if (discussion.value) {
-      discourseTopic.value = await loadSingleTopic(discussion.value);
+      try {
+        discourseTopic.value = await loadSingleTopic(discussion.value);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+    
+    if (props.space.additionalRawData?.boost?.enabled) {
```

Comment:
This global setting will hide previous proposals with boost, created when the space has enabled boost (but not currently)

Reply:
Ideally, boost setting should also be saved on proposal, like `privacy`

Reply:
Hmm right now it is enabled globally or disabled globally at space level on v1

Reply:
> This global setting will hide previous proposals with boost, created when the space has enabled boost (but not currently)

That's the expected behavior